CONTRIBUTING.md

```
@@ -0,0 +1,126 @@
+required to include 100% test coverage in order to be merged. Please
+don't hesitate to ask for help if you need one with the tests.
+
+Thanks a lot for your support.
+
+## Running tests
+
+First step is setting up development environment and installing all the
+required dependencies with:
+
+``` shell
+$ virtualenv -p python3 venv
```

markdownlint-cli2 (0.17.2)

Dollar signs used before commands without showing output
null

(MD014, commands-show-output)